templates/release-statefulset.yaml

```diff
@@ -135,6 +135,8 @@ spec:
           securityContext: {{- .Values.volumePermissions.containerSecurityContext | toYaml | nindent 12 }}
           {{- if .Values.volumePermissions.resources }}
           resources: {{- toYaml .Values.volumePermissions.resources | nindent 12 }}
+          {{- else if ne .Values.volumePermissions.resourcesPreset "none" }}
+          resources: {{- include "release.resources.preset" (dict "type" .Values.volumePermissions.resourcesPreset) | nindent 12 }}
```

Comment:
Not sure about that we should change the resources on the init-containers, in case of large and other options it will need a lot of resources. 
I think we need for init-containers split resource configuration to just specific predefined values, and not use presets.